tests/test_nbconflux.py

```diff
@@ -82,7 +82,8 @@ def test_post_to_confluence(notebook_path, page_url, server):
     server.add('POST', 'http://confluence.localhost/rest/api/content/12345/child/attachment/1/data')
     server.add('POST', 'http://confluence.localhost/rest/api/content/12345/child/attachment')
 
-    html, resources = nbconflux.notebook_to_page(notebook_path, page_url, 'fake-username', 'fake-pass')
+    html, resources = nbconflux.notebook_to_page(notebook_path, page_url, 'fake-username', 'fake-pass',
```

Comment:
The server mock response fixture up above might need to grow a couple more handlers for the additional labels so that the expected requests/responses line up properly for assertions down below. I see there's one POST handled for label. I suspect there should be 3 total now (nbconflux + the two extras).